tests/api.c

```diff
@@ -15772,11 +15772,13 @@ static int test_wc_PKCS7_EncodeSignedData(void)
     {
         int signedSz = 0, i;
         encodeSignedDataStream strm;
-        int numberOfChunkSizes = 4;
-        word32 chunkSizes[] = { 4080, 4096, 5000, 9999 };
+        static const int numberOfChunkSizes = 4;
```

Comment:
Why do these need to be `static`? I would think just `const` would be enough.

Reply:
moving them to the `rodata` segment adds bonus debugging of errant writes for free.